src/PRHardware.h

```diff
@@ -45,260 +45,7 @@ const int32_t FTDI_FT240X_PRODUCT_ID = 0x6015;
 
 //const int32_t FTDI_BUFFER_SIZE = 2048;
 const int32_t FTDI_BUFFER_SIZE = 8192;
```

Comment:
The `const` values below became public macros in `include/pinproc.h` so users of the library (like `pinproctest`) can make use of them.